### PR TITLE
OSDOCS-8517: 4.15 Cloud Compute Bug Batching

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-8517

Link to docs preview:
https://71878--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-bug-fixes

QE review:
N/A

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -1244,6 +1244,30 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
...
+
+* Previously, some conditions during the startup process of the Cluster Autoscaler Operator caused a lock that prevented the Operator from successfully starting and marking itself available. As a result, the cluster became degraded. The issue is resolved with this release. (link:https://issues.redhat.com/browse/OCPBUGS-18954[*OCPBUGS-18954*])
+
+* Previously, attempting to perform a Google Cloud Platform XPN internal cluster installation failed when control nodes were added to a second internal instance group. This bug has been fixed. (link:https://issues.redhat.com/browse/OCPBUGS-5755[*OCPBUGS-5755*])
+
+* Previously, the termination handler prematurely exited before marking a node for termination. This condition occured based on the timing of when the termination signal was received by the controller. With this release, the possibility of early termination is accounted for by introducing an additional check for termination. (link:https://issues.redhat.com/browse/OCPBUGS-2117[*OCPBUGS-2117*])
```
Comment (tmalove) [inline release_notes/ocp-4-15-release-notes.adoc:1269]:
```suggestion
* Previously, the termination handler prematurely exited before marking a node for termination. This condition occurred based on the timing of when the termination signal was received by the controller. With this release, the possibility of early termination is accounted for by introducing an additional check for termination. (link:https://issues.redhat.com/browse/OCPBUGS-2117[*OCPBUGS-2117*])
```

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -1244,6 +1244,30 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
...
+
+* Previously when deploying a hosted control plane (HCP) KubeVirt cluster the `--node-selector` command, the node selector was not applied to the `kubevirt-cloud-controller-manager` pods within the HCP namespace. Consequentially, you could not pin the entire HCP pods to specific nodes. With this update, this issue has been fixed. (link:https://issues.redhat.com/browse/OCPBUGS-27071[*OCPBUGS-27071*])
+
+* Previously, the default virtual machine (VM) type for the Microsoft Azure load balancer was changed from `Standard` to `VMSS`. Consequently, the service type load balancer could not attach standard VMs to load balancers. This update reverts these changes to the previous configuration to maintain compatibility with {product-title} deployments. As a result, load balancer attachments are now more consistent. (link:https://issues.redhat.com/browse/OCPBUGS-26210[*OCPBUGS-26210*])
+
+* Previously, deployments on {rh-openstack} nodes with additional ports with the `enable_port_security` field set to `false` were prevented from creating a LoadBalancer services. With this update, this issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-22246[*OCPBUGS-22246*])
```
[the remark this answers is not in the document]
Comment (obrown1205) [reply release_notes/ocp-4-15-release-notes.adoc:1255]: ty!

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -1244,6 +1244,30 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
...
+
+* Previously, in certain proxied environments, the Amazon Web Services (AWS) metadata service might not have been present on initial startup, and might have only been available shortly after startup. The kubelet hostname fetching did not account for this delay and, consequently, the node would fail to boot because it would not have a valid hostname. This update ensures that the hostname fetching script retries on failure for some time. As a result, inaccessibility of the metadata service is tolerated for a short period of time. (link:https://issues.redhat.com/browse/OCPBUGS-20369[*OCPBUGS-20369*])
+
+* In {product-title} version 4.14 and later, there is a known issue that causes installation of Azure Stack Hub to fail. Azure Stack Hub clusters that are upgraded to 4.14 or later might encounter load balancer configuration issues as nodes scale up or down. Installing or upgrading to 4.14 in Azure Stack Hub environments is not recommended until this issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-20213[*OCPBUGS-20213*])
```
[the remark this answers is not in the document]
Comment (obrown1205) [reply release_notes/ocp-4-15-release-notes.adoc:1263]: Good catch :) I think I should include the full name based on https://www.ibm.com/docs/en/ibm-style?topic=references-product-service-names#service-names

**Pull request merged**